CommonMark/CommonMarkAdditionalFeatures.cs

```diff
@@ -21,6 +21,26 @@ public enum CommonMarkAdditionalFeatures
         StrikethroughTilde = 1,
 
         /// <summary>
+        /// The parser will recognize syntax <c>~foo~</c> that will be rendered as <c>&lt;sub&gt;foo&lt;/sub&gt;</c>.
+        /// </summary>
+        SubscriptTilde = 2,
+
+        /// <summary>
+        /// The parser will recognize both <see cref="StrikethroughTilde"/> and <see cref="SubscriptTilde"/>.
+        /// </summary>
+        Tilde = 3,
+
+        /// <summary>
+        /// The parser will recognize syntax <c>^foo^</c> that will be rendered as <c>&lt;sup&gt;foo&lt;/sup&gt;</c>.
+        /// </summary>
+        SuperscriptCaret = 4,
+
+        /// <summary>
+        /// The parser will treat reference labels as case sensitive.
+        /// </summary>
+        RespectReferenceCase = 0x100,
```

Comment:
I'm all for consistency, but I wanted an explicitly distant value, and 256 wouldn't be very descriptive for the less fortunate ones who don't think in binary ;)